tests/build/build.go

```diff
@@ -71,6 +71,9 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Skip("Using private cluster (not reachable from Github), skipping...")
 			}
 
+			// Used for identifying related webhook on GitHub - in order to delete it
+			pacControllerHost = getPacControllerHost(f)
```

Comment:
we are duplicating this code here .. the getPacControllerHost functionality already been present few lines above and named as `osConsoleHost` .. maybe use that instead?

Reply:
@tisutisu I baked the openshift console host and app domain (used for detecting and deleting the PaC webhooks) right into the framework struct, so it's initialized only once per test run and can be used from any test. [PTAL](https://github.com/redhat-appstudio/e2e-tests/pull/1165/commits/73112be01525f9ec368efc73b7157bf2c0cd3267)